AcsfToolsCommands.php

```diff
@@ -148,6 +150,9 @@ public function sitesInfo() {
    *   Run cache clear on all sites with delay of 10 seconds between each site.
    * @usage drush acsf-tools-ml cron --use-https=1
    *   Run cron on all sites using secure url for URI.
+   * @usage drush acsf-tools-ml cron --use-https=1 --domain-containing=collection
+   *   Run cron on all sites using secure url and domain that contains the keyword factory for URI.
```

Comment:
It may be the good place to indicate that by default, it uses the first custom domain (the acsitefactory domain otherwise).

Reply:
added